### PR TITLE
Fix type warnings

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "polymer-analyzer": "2.0.0-alpha.18",
     "polymer-build": "^0.5.1",
     "polymer-project-config": "^1.0.0",
-    "polyserve": "0.16.0-prerelease.3",
+    "polyserve": "^0.16.0-prerelease.4",
```
Comment (justinfagnani) [inline package.json]: I think we want to pin exactly for now, would could have some breaking changes before release.

File: src/build/build.ts [hunk 2]
```diff
@@ -115,7 +115,7 @@ export async function build(options: BuildOptions, config: ProjectConfig): Promi
 
   // Once the unbundled build stream is complete, create a service worker for the build
   let unbundledPostProcessing = Promise.all([loadSWConfig, waitFor(unbundledPhase)]).then((results) => {
-    let swConfig: SWConfig = results[0];
+    let swConfig: SWConfig|undefined = results[0] || undefined;
```
Comment (justinfagnani) [inline src/build/build.ts]: what error were you seeing here?

Comment (justinfagnani) [inline src/build/build.ts]: Why weren't we seeing this locally or an the CI?

Comment (rictic) [inline src/build/build.ts]: Hm. Oh, I'm using a prerelease version of the compiler, I should switch back to stable.

File: package.json (hunk 1 above)
Comment (rictic) [inline package.json]: Done

**Comment by rictic** [comment]
Pinned, also fixed one other type temporarily

File: src/build/build.ts (hunk 2 above)
Comment (FredKSchott) [inline src/build/build.ts]: @rictic still needed after pinning?

Comment (rictic) [inline src/build/build.ts]: The stable compiler doesn't complain, but this code is more correct than before. `parsePreCacheConfig` returns `Promise<SWConfig|null>` and `addServiceWorker` wants `SWConfig|undefined`

**Comment by rictic** [comment]
I'm just gonna block this on https://github.com/PolymerLabs/polyserve/pull/136 because prerelease 5 has old type definitions.

**Pull request merged**